specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-11-01/machineLearningServices.json

```diff
@@ -0,0 +1,3060 @@
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the Machine Learning compute."
+        },
+        "createdOn": {
+          "type": "string",
+          "description": "The date and time when the compute was created.",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "modifiedOn": {
+          "type": "string",
+          "description": "The date and time when the compute was last modified.",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "resourceId": {
+          "type": "string",
+          "description": "ARM resource id of the underlying compute"
+        },
+        "provisioningErrors": {
+          "type": "array",
+          "description": "Errors during provisioning",
+          "items": {
+            "$ref": "#/definitions/MachineLearningServiceError"
+          },
+          "readOnly": true
+        },
+        "isAttachedCompute": {
+          "type": "boolean",
+          "description": "Indicating whether the compute was provisioned by user and brought from outside if true, or machine learning service provisioned it if false.",
+          "readOnly": true
+        }
+      },
+      "required": [
+        "computeType"
+      ]
+    },
+    "AKS": {
+      "description": "A Machine Learning compute based on AKS.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "type": "object",
+              "description": "AKS properties",
+              "properties": {
+                "clusterFqdn": {
+                  "description": "Cluster full qualified domain name",
+                  "type": "string"
+                },
+                "systemServices": {
+                  "type": "array",
+                  "description": "System services",
+                  "items": {
+                    "$ref": "#/definitions/SystemService"
+                  },
+                  "readOnly": true
+                },
+                "agentCount": {
+                  "description": "Number of agents",
+                  "type": "integer",
+                  "minimum": 1
+                },
+                "agentVMSize": {
+                  "description": "Agent virtual machine size",
+                  "type": "string"
+                },
+                "sslConfiguration": {
+                  "description": "SSL configuration",
+                  "$ref": "#/definitions/SslConfiguration"
+                },
+                "aksNetworkingConfiguration": {
+                  "description": "AKS networking configuration for vnet",
+                  "$ref": "#/definitions/AksNetworkingConfiguration"
+                }
+              }
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "AKS"
+    },
+    "AmlC
```

Comment:
I talked with Ahmet and Ahmet suggests not adding this. If needed, we can add it later and it will not be a breaking change. Please remove this property.